tests/nnstreamer_converter/unittest_converter.cc

```diff
@@ -1,737 +1,737 @@
-  ex = nnstreamer_converter_find ("python3");
-  ASSERT_NE (nullptr, ex);
-
-  EXPECT_EQ (FALSE, ex->get_out_config (caps, NULL));
-  g_usleep (100000);
-  gst_caps_unref (caps);
-}
-
-/**
- * @brief Test for python custom converter with invalid param
- */
-TEST (tensorConverterPython, invalidParam4_n)
-{
-  const gchar *root_path = g_getenv ("NNSTREAMER_SOURCE_ROOT_PATH");
-  const NNStreamerExternalConverter *ex;
-  void *py_core = NULL;
-  GstTensorsConfig config;
-  gchar *test_model;
-
-  gst_tensors_config_init (&config);
-  /** supposed to run test in build directory */
-  if (root_path == NULL)
-    root_path = "..";
-
-  ex = nnstreamer_converter_find ("python3");
-  ASSERT_NE (nullptr, ex);
-
-  test_model = g_build_filename (root_path, "tests", "test_models", "models",
-      "custom_converter_test.py", NULL);
-
-  EXPECT_EQ (0, ex->open (test_model, &py_core));
-  EXPECT_FALSE (ex->convert (NULL, &config, py_core));
-
-  ex->close (&py_core);
-  g_free (test_model);
-}
-
-/**
- * @brief Test for python custom converter with invalid param
- */
-TEST (tensorConverterPython, invalidParam5_n)
-{
-  const gchar *root_path = g_getenv ("NNSTREAMER_SOURCE_ROOT_PATH");
-  const NNStreamerExternalConverter *ex;
-  void *py_core = NULL;
-  gchar *test_model;
-  GstBuffer *buf;
-
-  /** supposed to run test in build directory */
-  if (root_path == NULL)
-    root_path = "..";
-
-  ex = nnstreamer_converter_find ("python3");
-  ASSERT_NE (nullptr, ex);
-
-  test_model = g_build_filename (root_path, "tests", "test_models", "models",
-      "custom_converter_test.py", NULL);
-  buf = gst_buffer_new ();
-  EXPECT_EQ (0, ex->open (test_model, &py_core));
-  EXPECT_FALSE (ex->convert (buf, NULL, py_core));
-
-  ex->close (&py_core);
-  g_free (test_model);
-  gst_buffer_unref (buf);
-}
-
-/**
- * @brief Test for python custom converter with invalid param
- */
-TEST (tensorConverterPython, invalidParam6_n)
-{
-  const gchar *root_path = g_getenv ("NNSTREAMER_SOURCE_ROOT_PATH");
-  const NNStreamerExternalConverter *ex;
-  void *py_core = NULL;
-  gchar *test_model;
-
-  /** supposed to run test in build directory */
-  if (root_path == NULL)
-    root_path = "..";
-
-  ex = nnstreamer_converter_find ("python3");
-  ASSERT_NE (nullptr, ex);
-
-  test_model = g_build_filename (root_path, "tests", "test_models", "models",
-      "invalid_class_custom_converter.py", NULL);
-  EXPECT_NE (0, ex->open (test_model, &py_core));
-
-  g_free (test_model);
-}
-
-/**
- * @brief Main GTest
- */
-int
-main (int argc, char **argv)
-{
-  int result = -1;
-
-  try {
-    testing::InitGoogleTest (&argc, argv);
-  } catch (...) {
-    g_warning ("catch 'testing::internal::<unnamed>::ClassUniqueToAlwaysTrue'");
-  }
-
-  gst_init (&argc, &argv);
-
-  try {
-    result = RUN_ALL_TESTS ();
-  } catch (...) {
-    g_warning ("catch `testing::internal::GoogleTestFailureException`");
-  }
-
-  return result;
-}
+/**
+ * @file        unittest_converter.cc
```

Comment:
What are the changes in this file?

Reply:
It's written in the PR description

Reply:
Such changes should be in another commit.